tests/generation/models/copynet_test.py

```diff
@@ -44,6 +44,13 @@ def test_model_can_train_with_amp(self):
             overrides="{'trainer.use_amp':true,'trainer.cuda_device':0}",
         )
 
+    def test_model_can_train_with_scheduled_sampling_ratio(self):
+        train_model_from_file(
+            self.param_file,
+            self.TEST_DIR,
+            overrides="{'model.scheduled_sampling_ratio':0.5}",
+        )
```

Comment:
> Added a simple test that the model can train with 0 < scheduled_sampling_ratio < 1. I wasn't really sure how to write a more significant test due to the randomness. Let me know if you have a different test in mind!

I think this is totally fine for a test. I just want to have coverage of the new lines of code to ensure that the model trains without error.